thorlcr/activities/rollup/throllup.cpp

```diff
@@ -43,7 +43,7 @@ CActivityBase *createDedupRollupActivityMaster(CMasterGraphElement *container)
     {
         if (TAKdedup == container->getKind())
         {
-            if (((IHThorDedupArg *)container->queryHelper())->compareAll())
+            if (((IHThorDedupArg *)container->queryHelper())->getFlags() & DFcompareall)
```

Comment:
It does because it makes the code easier to read/less likely to be wrong, and reduces the diff.